FAQ's.md

```diff
@@ -0,0 +1,30 @@
+**Q: What is ADSdroid?**
+
+A: It is an unofficial AllDataSheet Android Application. It gives her users the privilege of searching and downloading datasheets for electronic components in PDF format, for easier and faster access 
+
+**Q: Does it requires internet access?**
+
+A: Absolutely. It requires internet connection, for searching and also for downloading files in PDF format.
+
+**Q: How do I use it?**
+
+A: Very simple. A valid electrical terms should be entered, the name of the component (can be partial)
```

Comment:
The second sentence should start with `Just enter the name of the component (can be partial)` (the rest is OK)